shell/mixins/resource-fetch.js

```diff
@@ -67,6 +76,12 @@ export default {
       if (this.init && neu) {
         this.$fetch();
       }
+    },
+    resourceQuery: {
+      handler() {
+        this.$fetch();
```

Comment:
It can be, I wasn't too worried about to be honest because requests from the worker are already deduped and are pretty cheap regardless but a simple if statement could save us some clock-cycles.